README.md

```diff
@@ -21,29 +21,20 @@ The Azure Resource Manager (ARM) Tools for Visual Studio Code provides language
   - [Variable references](https://go.microsoft.com/fwlink/?LinkID=733960)
   - [resourceGroup() properties](https://azure.microsoft.com/en-us/documentation/articles/resource-group-template-functions/#resourcegroup)
   - [subscription() properties](https://azure.microsoft.com/en-us/documentation/articles/resource-group-template-functions/#subscription)
+  - [resourceId() function](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#resourceid)
   - Properties of references to variables that are objects
 
   ## Parameter files
 
   - Associate a parameter file with a template for full validation of both
   - Open associated parameter file from the template
   - Create a new parameter file from parameters defined within a template
```

Comment:
Do we need instructions for creating a new param file?  Should it be given better visibility in the readme?

Reply:
Agreed. I've added a little bit and also added a link to the [VS Code tutorial](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/quickstart-create-templates-use-visual-studio-code?tabs=CLI) which includes this functionality. I would be keen on considering another re-write of this readme that is more action > outcome oriented but will do so for a future release.